pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -942,6 +942,9 @@ func (gce *GCECloud) firewallObject(name, region, desc string, sourceRanges nets
 	if err != nil {
 		return nil, err
 	}
+	if len(hostTags) == 0 {
+		return nil, fmt.Errorf("Cannot create firewall without any tags found")
```

Comment:
can you make this something like "no node tags supplied and failed to parse the given lists of hosts for tags. Not creating firewall rule."